### PR TITLE
Each e-mail contains schema.org metadata about the booking event.

### PR DESCRIPTION
**Pull request by jasiek**
A button in Gmail that provides "add to calendar" functionality from email list view using schema.org.

### CODE REVIEW

**Comment by tonyduarte** [comment]
hey Jan, what does this address? thanks.

**Comment by jasiek** [comment]
You know, how in GMail you will sometimes have a hint above an e-mail (https://support.google.com/calendar/answer/47802?hl=en) that prompts you to add it to the calendar? Me and @matharden thought it would be useful to have this for those of our clients who use GMail and other "smart" email providers.

**Comment by SteveMarshall** [comment]
@jan Could you rebase to describe that reasoning in the commit message? (And, ultimately, the PR) That way, anyone who wants to see where this code came from gets the same description without too much effort.

File: app/views/mailers_common/_booking_confirmation.html.haml [hunk 1]
```diff
@@ -114,3 +114,6 @@
 
 %p
   %small Visit ID: #{@visit.visit_id}
+
+%script{type: 'application/ld+json'}
```
Comment (matharden) [inline app/views/mailers_common/_booking_confirmation.html.haml]: I'm concerned about this. This puts a `script` tag in the email. I thought webmail clients strip these from email as a security measure. Does this work?

Comment (jasiek) [inline app/views/mailers_common/_booking_confirmation.html.haml]: That could be the case. Alternatively, we can use HTML tags to convey the same information.

Comment (SteveMarshall) [inline app/views/mailers_common/_booking_confirmation.html.haml]: We should do this in HTML if we can, I think. It makes our markup more semantic and means the metadata is then a first-class citizen and less likely to be subject to the sort of scrutiny that JavaScript is (by less modern mail clients, anyway), as @mat mentions.

Comment (matharden) [inline app/views/mailers_common/_booking_confirmation.html.haml]: Defo go for HTML!

**Comment by SteveMarshall** [comment]
Any chance we can get this done and merged this week?

**Comment by threedaymonk** [comment]
Closing stale PR as discussed. This feature would be nice to have, however.

**Pull request closed without merging**